rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -261,6 +295,15 @@ impl IdCompressor {
         }
     }
 
+    /// Normalizes an op space ID to this session's session space.
```

Comment:
It's probably worth noting the difference between an originator and a creator here.

Reply:
And on the other variant of normalize to session space as well